staging/src/kubevirt.io/client-go/kubecli/version.go

```diff
@@ -54,7 +55,12 @@ func (v *ServerVersion) Get() (*version.Info, error) {
 
 	var group metav1.APIGroup
 	// First, find out which version to query
-	uri := ApiGroupName
+	parsedUrl, err := url.Parse(v.restClient.Get().URL().Path)
+	if err != nil {
+		return nil, err
+	}
+	sBasePath := strings.Replace(parsedUrl.Path, "/apis/"+v.restClient.APIVersion().String(), "", 1)
+	uri := sBasePath + ApiGroupName
```

Comment:
Probably, we can make it even simpler:
```suggestion
	u, err := url.Parse(v.config.Host)
	if err != nil {
		return nil, err
	}
	uri := path.Join(u.Path, ApiGroupName)
```
I think this should work without `Replace` as `v.config.Host` contains the initial cluster URL.

Then further you can do:
```go
uri = fmt.Sprintf("%s/apis/%s/version", u.Path, group.PreferredVersion.GroupVersion)
```

Reply:
I see. I actually now also changed in on vm and vmi.go the same way. I had to add the v.config for version which was not existing yet